### PR TITLE
Feature/school library UI

### PR DESCRIPTION
**Pull request by topeogunleye**
### Added Changes
Added School Library UI

### CODE REVIEW

File: app.rb [hunk 1]
```diff
@@ -0,0 +1,188 @@
...
+    subject = gets.chomp
+    puts 'What is the age of the teacher?'
+    age = gets.chomp.to_i
+    puts 'What is the name of the teacher?'
+    name = gets.chomp
+    puts 'Does the teacher have permission to use the library services [Y/N]: ?'
+    permission = choice_permission
+    teacher = Teacher.new(subject, age, name, parent_permission: permission)
+    people_store(teacher)
+  end
```
Comment (DeliceLydia) [inline app.rb]: - [ ] When I am creating a person 1 should be a `student` and 2 should be a `teacher` currently when I press 1 it displays questions of the teacher and vice versa please address that
- [ ] Remember when  you are creating a teacher  you will first ask his/her `specialization` then `name`, `age`

File: app.rb [hunk 2]
```diff
@@ -0,0 +1,188 @@
...
+    classroom = gets.chomp
+    puts 'What is the age of the student?'
+    age = gets.chomp.to_i
+    puts 'What is the name of the student?'
+    name = gets.chomp
+    puts 'Does the student have permission to use the library services?'
+    permission = choice_permission
+    student = Student.new(classroom, age, name, parent_permission: permission)
+    people_store(student)
+  end
```
Comment (DeliceLydia) [inline app.rb]: - [ ] I suggest that when you create a person you just ask their `names` and `ages` it's not necessary to ask other questions like `What classroom is the student?`
- [ ] Remember to add a message like "Person created successfully" when it is successful currently I am not able to create a person to be able to create a book or interact with the rest of the application

**Review by DeliceLydia — CHANGES_REQUESTED** [review]
## Hi @topeogunleye, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there! Kudos on
- Awesome README
- Following git workflow


## Required Changes ♻️
_Check the comments under the review._



### Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)

File: app.rb [hunk 3]
```diff
@@ -0,0 +1,192 @@
...
+
+def list_people
+  @people_arr.each { |person| puts "[#{person.class}]: Name: #{person.name}, ID: #{person.id} AGE: #{person.age}" }
+  main
+end
+
+def choice_permission
+  print 'Has parent permission? [Y/N]: '
+  parent_permission = gets.chomp
+  print 'Enter the classroom: '
```
Comment (rukundoeric) [inline app.rb:67]: - [ ] <b>REQUIRED:</b> As mentioned in the previous review, classroom is not required in creating people, student or teacher. Kindly remove it.
scrnli_5_14_2022_4-08-49 PM

File: app.rb [hunk 4]
```diff
@@ -0,0 +1,192 @@
...
+  main
+end
+
+def list_people
+  @people_arr.each { |person| puts "[#{person.class}]: Name: #{person.name}, ID: #{person.id} AGE: #{person.age}" }
+  main
+end
+
+def choice_permission
+  print 'Has parent permission? [Y/N]: '
```
Comment (rukundoeric) [inline app.rb:65]: - [ ] <b>REQUIRED:</b> Also, `Has parent permission? [Y/N]:` should only be asked if the person is a student. Teachers are not required to have parents's permission.
scrnli_5_14_2022_4-11-43 PM

**Review by rukundoeric — APPROVED** [review]
Hi @topeogunleye, 

_Great job so far 👏 💯 👍. You've done well implementing some of the requested changes. However, there are still some which aren't addressed yet. Please, check the comments below._
## Highlights
  - Follows git workflow.  💯
  - Codebase looks clean and professional. 💯 
  - Addressed some of the issues raised from the previous review. 💯 

## Required Changes ♻️
  - [ ] Check the comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
  
  - N/A 

Cheers and Happy coding!👏👏👏

__Feel free to leave any questions or comments in the PR thread if something is not 100% clear.__

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
> As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: app.rb (hunk 3 above)
Comment (julie-ify) [reply app.rb:67]: This has been resolved 👍

File: app.rb (hunk 4 above)
Comment (julie-ify) [reply app.rb:65]: This has been resolved 👍

**Review by julie-ify — APPROVED** [review]
Hi @topeogunleye, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

# Status: Approved ✔️ 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**